src/devtools/server/actors/highlighters/box-model.js

```diff
@@ -705,6 +706,7 @@ class BoxModelHighlighter extends AutoRefreshHighlighter {
    */
   _updateGuide(side, point = -1) {
     const guide = this.getElement("guide-" + side);
+    const scale = getDevicePixelRatio();
```

Comment:
Similarly, a `scalePoint` could be useful but inlining the calc after this line would be okay too